buildpacks/maven/tests/misc_tests.rs

```diff
@@ -0,0 +1,153 @@
+use indoc::indoc;
+use libcnb_test::{
+    assert_contains, assert_not_contains, BuildpackReference, PackResult, TestConfig, TestRunner,
+};
+use std::fs;
+use std::os::unix::fs::PermissionsExt;
+
+#[test]
+fn mvnw_executable_bit() {
+    TestRunner::default().run_test(
+        default_config().app_dir_preprocessor(|dir| {
+            fs::set_permissions(dir.join("mvnw"), fs::Permissions::from_mode(0o444)).unwrap();
+        }),
+        |context| {
+            // Assert the build completed successfully, even with a missing executable bit on
+            // the Maven Wrapper executable.
+            assert_contains!(context.pack_stdout, "Successfully built image");
```

Comment:
(Observation only) Isn't this redundant given (a) it's more the non-zero exit code that's key here, (b) this string could change in future pack/lifecycle versions?